carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcasterConfig.hpp

```diff
@@ -29,6 +29,7 @@ namespace carma_wm_ctrl
   {
     int ack_pub_times = 1; // The number of times it publishes Geofence Acknowledgement.
     double max_lane_width = 4.0; // Max lane width in meters within which geofence points are associated to a lanelet as those points are guaranteed to apply to a single lane
+    double tcr_bbox_expansion_meters = 50.0; // How much in meters to expand the TCR Bounding Box
```

Comment:
Can we add to the stream operator as well?

Reply:
Added